script.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable max-len */
```

Comment:
Great job team 🤝

- [ ] Kindly remove these linter disabling lines of code from your project. Linter checks help us write more professional code.